src/net/sourceforge/kolmafia/request/AccountRequest.java

```diff
@@ -205,6 +205,11 @@ private static void parseCombatOptions(final String responseText) {
     KoLCharacter.setAutoAttackAction(autoAttackAction);
   }
 
+  private static final Pattern TIMEZONE_PATTERN =
+      Pattern.compile("<select name=\"timezone\">.*?</select>", Pattern.DOTALL);
+  private static final Pattern SELECTED_PATTERN_2 =
+      Pattern.compile("<option selected='selected'>(.*?)</option>");
```

Comment:
```suggestion
      Pattern.compile("<option selected='selected'>([^<]+)</option>");
```
might be slightly faster. Minor, though.